addons/sale_project/models/project.py

```diff
@@ -267,7 +267,7 @@ def get_sale_items_data(self, domain=None, offset=0, limit=None, with_action=Tru
         def get_action(sol_id):
             """ Return the action vals to call it in frontend if the user can access to the SO related """
             action, res_id = action_per_sol.get(sol_id, (None, None))
-            return {'action': action, 'res_id': res_id, 'additional_context': json.dumps({'active_id': sol_id, 'default_project_id': self.id})} if action else {}
+            return {'action': action, 'res_id': res_id, 'additional_context': json.dumps({'active_id': sol_id, 'default_project_id': self.id, 'graph_groupbys': ['date:month', 'employee_id']})} if action else {}
```

Comment:
Cannot we change the graph view or given another graph view with the field in the row that we want? 🤔